### ISSUE TITLE
Hello, contact

### ISSUE THREAD

**Issue by schonorio** [andrewAsdf/pokerbot#1]
Hi Andrew,

I am starting studing about poker bots and found you code. Can you explain to me how it works?

**Comment by b-thebest** [comment]
Yes, Please write some document about basic structure and flow of you code. It will be great help for students.

Thanks
